package.json

```diff
@@ -59,6 +59,7 @@
     "eslint": "^4.2.0",
     "eslint-config-mourner": "^2.0.1",
     "express": "^4.13.4",
+    "geojson": "^0.5.0",
```

Comment:
This should be added via `yarn add geojson`, right now the yarn.lock is not updated

Reply:
Got it